### DISCUSSION TITLE
Noi Ask doesn't work with ChatGPT since switching to team version but works OK with Gemini and Claude. Any advice on fixing this? Thanks!

### DISCUSSION THREAD

**Discussion by TyrealQ** [lencx/Noi#149] [General]
Noi Ask doesn't work with ChatGPT since switching to team version but works OK with Gemini and Claude. Any advice on fixing this? Thanks!

**Comment by joaquincodigo** [comment]
I have the same problem. They recommend updating the manifest.json and main.js from the "Noi Ask" repo. I did but the problem persists. Its a shame, it throws all away all the effort they made to create the the software. Its useless to me :(

**Comment by markflores1777-ctrl** [comment]
What is lawful money and what do you get when you issue to the Treasury a lawful instrument demanding it be redeemed in lawful money?
